apps/inviwo/packaging/packaging.cmake

```diff
@@ -114,6 +114,31 @@ elseif(APPLE)
     configure_file("${IVW_ROOT_DIR}/cmake/deploy-osx.cmake.in" "${PROJECT_BINARY_DIR}/deploy-osx.cmake" @ONLY)
     set(CPACK_PRE_BUILD_SCRIPTS "${PROJECT_BINARY_DIR}/deploy-osx.cmake")
 else()
+    file(GENERATE OUTPUT ${PROJECT_BINARY_DIR}/${IVW_PACKAGE_SELECT_APP}.desktop CONTENT
+      "[Desktop Entry]"
+      "Type=Application"
+      "Name=${IVW_PACKAGE_SELECT_APP}"
+      "Exec=${IVW_PACKAGE_SELECT_APP}"
+      "Icon=${IVW_PACKAGE_SELECT_APP}"
+      "Categories=Science;"
+    )
+    install(FILES ${PROJECT_BINARY_DIR}/${IVW_PACKAGE_SELECT_APP}.desktop 
+            DESTINATION share/applications 
+            COMPONENT Application
+    )
+    install(FILES ${IVW_ROOT_DIR}/resources/inviwo/inviwo-logo-512px.png  

```

Comment:
Any idea whether SVG icons are also supported?

Reply:
I believe so, but the first goal is to make it work.
One can dive into it here:
https://specifications.freedesktop.org/icon-theme-spec/icon-theme-spec-latest.html